### ISSUE TITLE
Gulp 'task finished' is never printed out w/ less compilation

### ISSUE THREAD

**Issue by perlun** [floatdrop/gulp-watch#38]
Hi,

I have a gulpfile.coffee (using the CoffeeScript plugin) that looks like this:

```
gulp.task 'less-styles', () ->
  watch(glob: [
      'src/styles/*.less',
      'src/styles/font-awesome/font-awesome.less'
    ],
    (files) ->
      files.pipe less(
        paths: [pathModule.join(__dirname, 'less', 'includes')]
      ).on 'error', notify.onError
        message: 'Error: <%= error.message %>',
        title: 'Error in styles'
      .on 'finish', () ->
        console.log "FOO!"
      .pipe gulp.dest './ui/styles'
  )
```

The problem is that when the file is changes, the task runs (and the `less` file gets recompiled). **But**, gulp doesn't print out its usual, colorful message. :cry: I assume this is since the task is never finished.

The 'finish' even runs though, so I could probably hook something up, but I'm unsure what `this` is in that case. And also, why is that needed?

In short: _How do you use gulp-watch and still get "task finished" messages being printed out?_

Maybe I'm trying to do things the wrong way, I'm a gulp n00b (but have managed to convert our old build pipeline to it, except that it e.g. doesn't detect files being added... :smile: Which is why I want to use gulp-watch)
